src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1499,6 +1499,41 @@ public virtual BadRequestObjectResult BadRequest(ModelStateDictionary modelState
         }
 
         /// <summary>
+        /// Creates an <see cref="UnprocessableEntityResult"/> that produces a <see cref="StatusCodes.Status422UnprocessableEntity"/> response.
+        /// </summary>
+        /// <returns>The created <see cref="UnprocessableEntityResult"/> for the response.</returns>
+        [NonAction]
+        public virtual UnprocessableEntityResult UnprocessableEntity()
```

Comment:
`public virtual UnprocessableEntityResult UnprocessableEntity() => new UnprocessableEntityResult()` for 💯 points

Reply:
There are many other methods like this in the same class, it it worth the inconsistency?

Reply:
Let's leave this for #6864 :smile: